interface/main/finder/dynamic_finder.php

```diff
@@ -337,33 +339,43 @@ function persistCriteria(el, e) {
-        <br />
-        <div id="dynamic"><!-- TBD: id seems unused, is this div required? -->
-            <!-- Class "display" is defined in demo_table.css -->
-            <table class="table table-borderless display" id="pt_table">
-                <thead>
-                    <tr id="advanced_search" class="hideaway" style="display: none;">
-                        <?php echo $header0; ?>
-                    </tr>
-                    <tr class="head">
-                        <?php echo $header; ?>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <!-- Class "dataTables_empty" is defined in jquery.dataTables.css -->
-                        <td class="dataTables_empty" colspan="<?php echo attr($colcount); ?>">...</td>
-                    </tr>
-                </tbody>
-            </table>
+            <?php if (AclMain::aclCheckCore('patients', 'demo', '', array('write','addonly'))) { ?>
+                <button id="create_patient_btn1" class="btn btn-primary btn-add" onclick="top.restoreSession();top.RTop.location = '<?php echo $web_root ?>/interface/new/new.php'"><?php echo xlt('Add New Patient'); ?></button>
+            <?php } ?>
+            <div class="jumbotron mt-3 p-4">
+                <div id="dynamic"><!-- TBD: id seems unused, is this div required? -->
+                    <!-- Class "display" is defined in demo_table.css -->
+                    <div class="table-responsive">
+                        <table class="table" cellpadding="0" cellspacing="0" class="border-0 display" id="pt_table">
+                            <thead>
+                                <tr id="advanced_search" class="hideaway"  style="display: none;">
+                                    <?php echo $header0; ?>
+                                </tr>
+                                <tr class="table-primary">
+                                    <?php echo $header; ?>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <!-- Class "dataTables_empty" is defined in jquery.dataTables.css -->
+                                    <td class="dataTables_empty" colspan="<?php echo attr($colcount); ?>">...</td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
+            <!-- form used to open a new top level window when a patient row is clicked -->
+            <form name='fnew' method='post' target='_blank' action='../main_screen.php?auth=login&site=<?php echo attr_url($_SESSION['site_id']); ?>'>
+                <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+                <input type='hidden' name='patientID' value='0'/>
+            </form>
```

Comment:
This code block (above 5 lines) is duplicated below.

Reply:
@stu01509 , just let me know when above issue is addressed and then I'll bring this in.